include/afl-fuzz.h

```diff
@@ -534,6 +534,7 @@ typedef struct afl_state {
 
 #define N_FUZZ_SIZE (1 << 21)
   u32 *n_fuzz, *n_fuzz_tmp;
+  u64 time_to_next;

```

Comment:
Seems like this corresponds to the pre-reset estimate for the time until the next discovery. Correct?

Reply:
Yes.